media-libs/libvmaf/libvmaf-2.3.0.ebuild

```diff
@@ -21,6 +21,8 @@ fi
 LICENSE="BSD-2-with-patent"
 SLOT="0"
 
+RESTRICT="test"
+
```

Comment:
We add a reason for disabling tests by adding a comment on top of it, preferably with an upstream issue so we can track when it gets fixed.

Reply:
The related bug is this one https://bugs.gentoo.org/835471, some paths are hardcoded and don't work. @thesamesam prefers that I look into it first. Although I cannot seem to find where the test executables are produced to fix this.

Reply:
Okay in the end it's an upstream issue, I opened a bug report https://github.com/Netflix/vmaf/issues/998

Should I disable tests for now or leave it be ? The library itself works for our use case anyway.

Reply:
Please add a link to that issue above the restriction